dev_guide/new_app.adoc

```diff
@@ -389,6 +389,16 @@ image stream is created for that image as well.
 a|`*DeploymentConfig*`
 a|A `*DeploymentConfig*` is created either to deploy the output of a build, or a
 specified image.
+ifdef::openshift-origin[]
+The `new-app` command creates link:volumes.html[*EmptyDir* volumes] for all
```

Comment:
The original line here said "OpenShift will create...", changed it to be more specific that it's a result of the new-app process.